admin_manual/configuration_files/external_storage/smb.rst

```diff
@@ -5,12 +5,12 @@ SMB/CIFS
 ownCloud can connect to Windows file servers or other SMB-compatible servers
 with the SMB/CIFS backend.
 
-.. note:: The SMB/CIFS backend requires ``smbclient`` or 
-   the PHP smbclient module to be installed on the ownCloud server. The PHP 
-   smbclient module is preferred, but either will work. These 
-   should be included in any Linux distribution. (See `PECL smbclient 
-   <https://pecl.php.net/package/smbclient>`_ if your distro does not include 
-   them.)
+.. note:: The SMB/CIFS backend requires the PHP smbclient module to be installed on the ownCloud server. This should be included in any Linux distribution; on Debian, Ubuntu, CentOS, and Fedora it is ``php-smbclient``. See `eduardok/libsmbclient-php <https://github.com/eduardok/libsmbclient-php>`_ if your distribution does not include it; this provides source archives and binary packages. 
```

Comment:
source archives and binary packages -> source archives and instructions how to install binary packages